test/unit/transport_test.rb

```diff
@@ -0,0 +1,99 @@
+# encoding: utf-8
+
+require 'helper'
+
+describe Inspec::Transport do
+  describe '#inspec transport' do
+    let(:transport) { Inspec::Transport.new({ backend: 'mock' }) }
+
+    describe 'mock transport initialized' do
+      it 'return mock transport on initialize' do
+        transport.must_be_kind_of Inspec::Transport
+        transport.conn.must_be_kind_of Train::Transports::Mock::Connection
+      end
+    end
+
+    describe 'create train connection' do
+      it 'return new train transport' do
+        trans = transport
```

Comment:
What's the purpose of assigning this to a new variable? This happens throughout the tests, so I'm not going to comment on all of them, but I'm curious as to why it's necessary.